src/components/Modal/Modal.tsx

```diff
@@ -107,7 +108,7 @@ class Modal extends PureComponent {
             <Button
               styled={false}
               className="absolute top-0 right-0 -mt-4 -mr-4"
-              onMouseUp={this.gracefulClose}
+              onMouseUp={() => this.gracefulClose}
```

Comment:
This just returns the function, it does not call it :) But I'll go ahead and fix it, no worries about it!

Reply:
Turns out, this button was never shown due to a CSS bug. So I removed it completely :)